mlrun/api/crud/client_spec.py

```diff
@@ -26,6 +26,9 @@ def get_client_spec(self):
             dask_kfp_image=config.dask_kfp_image,
             api_url=config.httpdb.api_url,
             nuclio_version=self._resolve_nuclio_version(),
+            valid_function_priority_class_names=config.valid_function_priority_class_names.split(
+                ","
+            ),
```

Comment:
1. The "parsing" logic (doing `.split()`) shouldn't sit here, cause with that if somewhere else in the code we'll use the value you'll have to duplicate the logic, should be in a method in the Config class.
2. The value of `valid_function_priority_class_names` should be a comma separated string, here you're returning a list, and assign it on the httpdb.py side, which can cause to weird bugs.

Reply:
After doing what I suggested above, you actually not gonna use the method from the Config class here (cause you want the string not the list) - but, although this config value does not have a default, I think you should send it only if it's set